src/actions/init.ts

```diff
@@ -0,0 +1,35 @@
+import * as gitClone from 'git-clone';
+import * as rimraf from 'rimraf';
+import * as npm from 'npm';
+import chalk from 'chalk';
+
+const defaultProjectName = 'biotope-boilerplate';
+export const registerInit = commander => commander
+    .command('init [name]')
+    .action((name = defaultProjectName) => {
+        console.log(chalk.green('========================================'));
+        console.log(chalk.green('======== Pulling repository 🏎️ ========='));
+        console.log(chalk.green('========================================'));
+        gitClone(
+            'https://github.com/biotope/biotope-boilerplate.git',
+            name,
```

Comment:
Is it possible to omit the name and add a default name?

Reply:
name IS optional. Default name is 'biotope-boilerplate'